src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -29,6 +30,7 @@ public class CheckResult : IOperationStatus
         /// null if unavailable.  
         /// This is only valid when <see cref="IsSuccess"/> is true.
         /// </summary>
+        [SuppressMessage("Usage", "CA2227: Collection properties should be read only", Justification = "n/a")]

```

Comment:
This probably should be readonly, actually. (or at least `internal set`)

Reply:
Works fine